Sources/TuistCore/Analytics/CommandEventGraph.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Graph to be sent with the command event
+public struct CommandEventGraph: Codable, Equatable {
```

Comment:
What about taking the opportunity to start aligning the terminology with the product concept of "Run". For example, I'd name this one `RunGraph`. And I'd do the same with others (e.g. `RunCacheTargetMetadata`)